docs/start/authentication/graphql-storefront.mdx

```diff
@@ -66,17 +66,226 @@ content-type: application/json
 </Tab>
 </Tabs>
 
+#### Customer access tokens
+
+A customer access token is unique to an individual user's account because it represents an authenticated storefront session for GraphQL requests. You can obtain and use a customer access token only for server-to-server requests. Therefore, you must use the customer access token with a regular storefront token. A customer access token becomes invalid on all devices when you log out of a single device. 
+
+<Callout type="warning">
+Do not use this token for browser-side or client-side requests. Browser-side requests require using the [customer impersonation token](#customer-impersonation-tokens) or the `SHOP_TOKEN` response cookie.
```

Comment:
customer impersonation token is not for browser-side requests! https://github.com/bigcommerce/docs/pull/365#discussion_r1686470808

also lower in the docs you have: 
``` The API will reject any customer impersonation token-authenticated requests that originate from a web browser and are not proxied. Use these tokens in NextJS applications; NextJS proxies requests by default. ```

Reply:
Thank you for catching this @kskomro . I removed the sentence.

Reply:
I think you removed the wrong one ;)
> The API will reject any customer impersonation token-authenticated requests that originate from a web browser and are not proxied. 
that was correct one and I used it to tell you that customer impersonation token is NOT allowed for browser-side requests.

Reply:
> Browser-side requests require using the [customer impersonation token]

this is not true.